src/sentry/dynamic_sampling/tasks/recalibrate_orgs.py

```diff
@@ -141,3 +157,74 @@ def recalibrate_org(org_id: int, total: int, indexed: int) -> None:
 
     # At the end we set the adjusted factor.
     set_guarded_adjusted_factor(org_id, adjusted_factor)
+
+
+@instrumented_task(
+    name="sentry.dynamic_sampling.tasks.recalibrate_projects_batch",
+    queue="dynamicsampling",
+    default_retry_delay=5,
+    max_retries=5,
+    soft_time_limit=25 * 60,
```

Comment:
`soft_time_limit` should be lower than `time_limit`

Reply:
Agreed. I'll add this to our milestone, we need to clean up all the tasks and determine better time limits.